src/Pages/Authentication/login.js

```diff
@@ -51,17 +51,28 @@ const Login = () => {
             }
           />
         </div>
-        <div className="input-box">
-          <input
-            type="Submit"
-            value="Login Now"
-            onClick={() => submitHandler()}
-          />
-        </div>
-        <div className="input-box">
-          <input type="Submit" value="Test Login" onClick={() => testlogin()} />
+        <div style={{ display: "flex", gap: "1rem" }}>
+          <div>
+            <input
+              type="Submit"
+              value="Login Now"
+              className="button btn-primary"
+              onClick={() => submitHandler()}
+            />
+          </div>
+          <div>
+            <input
+              type="Submit"
+              value="Test Login"
+              className="button btn-primary"
+              onClick={() => testlogin()}
```

Comment:
you don't need to write arrow function to call the function you call with its name only without ( () ) parentheses.

Reply:
Thankyou. Will do that.